R/apply_delay.R

```diff
@@ -1,7 +1,12 @@
 #' Apply the delay to generate a point nowcast
 #' Generate a point estimate of a completed reporting square (or rectangle)
 #'   from a reporting triangle that we want to complete with a nowcast and a
-#'   delay PMF. This code was adapted from code written (under an MIT license)
+#'   delay PMF. Each element is computed by taking the product of the expected
+#'   number of total cases assigned to a reference time $t$ and the proportion
+#'   of those cases reported on delay $d$. The formula to obtain the expected
+#'   number of total cases as a function of the reporting delay and previous
+#'   observations was derived elsewhere.
```

Comment:
is this meant as a reference? I think its a bit clunky. Shall we just have an issue to remember to remove it and add the reference when it is available + also improve these docs with a bit of motivation to help readers who don't want to follow the reference?

Reply:
Yeah I think that is a good idea -- I see the above comment about placeholders, so am guessing we just want to flag these places where wed update in the issue?

Reply:
New issue: https://github.com/epinowcast/baselinenowcast/issues/70